include/sir/config.h

```diff
@@ -69,10 +69,33 @@
 # endif
 
 /**
- * Define to use <CR><LF> line endings, otherwise use <LF>.
+ * Define the system carriage return (CR) character.
```

Comment:
`The carriage return (CR) character to use in the end of line sequence when SIR_USE_EOL_CRLF is defined.`

Reply:
Done.